Meta/Lagom/Tools/CodeGenerators/LibTimeZone/GenerateTimeZoneData.cpp

```diff
@@ -112,11 +112,17 @@ template<>
 struct AK::Formatter<DaylightSavingsOffset> : Formatter<FormatString> {
     ErrorOr<void> format(FormatBuilder& builder, DaylightSavingsOffset const& dst_offset)
     {
+        auto format_time = [&](auto year) -> String {
+            if (year == NumericLimits<u16>::max())
+                return "max_year"sv;
```

Comment:
Are you sure it's max? Since we do `+ 1` below I think it has to be `NumericLimits<u16>::max() + 1`

Reply:
Yeah i just noticed that too - I think I'm gonna make this just an Optional instead of ::max

Reply:
Yeah looking at the generated output I see just one `max_year` but still a bunch `AK::Time::from_timestamp(65536, 1, ...)`

Reply:
With it fixed it seems to speed up compilation of the generated cpp file from ~250ms to ~170 ms, although it is a little bit noisy